rotkehlchen/data/importer.py

```diff
@@ -469,3 +480,269 @@ def import_cryptocom_csv(self, filepath: Path) -> Tuple[bool, str]:
+
+        if len(csv_row['Confirmed At']) != 0:
+            timestamp = deserialize_timestamp_from_date(
+                date=csv_row['Confirmed At'],
+                formatstr='%Y-%m-%d %H:%M:%S',
+                location='BlockFi',
+            )
+        else:
+            raise DeserializationError('Action not confirmed yet')
+
+        asset = symbol_to_asset_or_token(csv_row['Cryptocurrency'])
+        amount = deserialize_asset_amount_force_positive(csv_row['Amount'])
+        entry_type = csv_row['Transaction Type']
+        # BlocFI doesn't provide information about fees
+        fee = Fee(ZERO)
+        fee_asset = A_USD  # Can be whatever
+
+        if entry_type in ('Deposit', 'Wire Deposit', 'ACH Deposit'):
+            asset_movement = AssetMovement(
+                location=Location.BLOCKFI,
+                category=AssetMovementCategory.DEPOSIT,
+                address=None,
+                transaction_id=None,
+                timestamp=timestamp,
+                asset=asset,
+                amount=amount,
+                fee=fee,
+                fee_asset=fee_asset,
+                link='',
+            )
+            self.db.add_asset_movements([asset_movement])
+        elif entry_type in ('Withdrawal', 'Wire Withdrawal', 'ACH Withdrawal'):
+            asset_movement = AssetMovement(
+                location=Location.BLOCKFI,
+                category=AssetMovementCategory.WITHDRAWAL,
+                address=None,
+                transaction_id=None,
+                timestamp=timestamp,
+                asset=asset,
+                amount=amount,
+                fee=fee,
+                fee_asset=fee_asset,
+                link='',
+            )
+            self.db.add_asset_movements([asset_movement])
+        elif entry_type == 'Withdrawal Fee':
+            action = LedgerAction(
+                identifier=0,  # whatever is not used at insertion
+                timestamp=timestamp,
+                action_type=LedgerActionType.EXPENSE,
+                location=Location.BLOCKFI,
+                amount=amount,
+                asset=asset,
+                rate=None,
+                rate_asset=None,
+                link=None,
+                notes=f'{entry_type} from BlockFi',
+            )
+            self.db_ledger.add_ledger_action(action)
+        elif entry_type in ('Interest Payment', 'Bonus Payment', 'Referral Bonus'):
+            action = LedgerAction(
+                identifier=0,  # whatever is not used at insertion
+                timestamp=timestamp,
+                action_type=LedgerActionType.INCOME,
+                location=Location.BLOCKFI,
+                amount=amount,
+                asset=asset,
+                rate=None,
+                rate_asset=None,
+                link=None,
+                notes=f'{entry_type} from BlockFi',
+            )
+            self.db_ledger.add_ledger_action(action)
+        elif entry_type == 'Trade':
+            pass
```

Comment:
why?

Reply:
BlockFi provides two files, the one imported in this method contains partial information about trades and you have to link the sell amount with the bought amount since there is no relation between them (they appear in different rows). The other file provides more useful information for the same trades